java/mtsj/pom.xml

```diff
@@ -155,13 +169,15 @@
       </plugin>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.22.0</version>
+        <version>3.0.0-M4</version>
```

Comment:
interesting. Is that now required? Do we also need to update this in `devon4j` archetype? Is the milestone version already stable or do we have to expect other bugs?

Reply:
I just increase the version because the 2.22.0 was not executing any test. I think devon4j archetype is using the 3.0.0-M3. I just updated to the latest one.